sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/NuGetMetadataReference.cs

```diff
@@ -56,20 +50,6 @@ internal static class NuGetMetadataReference
         private static readonly PackageManager packageManager =
             new PackageManager(CreatePackageRepository(), PackagesFolderRelativePath);
 
-        private static Lazy<Dictionary<string, MetadataReference[]>> CreateLazy(string packageId, params string[] versions)
-        {
-            foreach (var packageVersion in versions)
-            {
-                allNuGets.Add((packageId, packageVersion));
-            }
-
-            return new Lazy<Dictionary<string, MetadataReference[]>>(Create, LazyThreadSafetyMode.PublicationOnly);
-
-            Dictionary<string, MetadataReference[]> Create() =>
-                versions.Select(v => new { version = v, references = CreateReferences(packageId, v) })
-                    .ToDictionary(x => x.version, x => x.references);
-        }
-
```

Comment:
The lazy dictionaries by package are no longer required. We could adding simpler caching for MetadataReferences if it turns out to be necessary.